src/singolo.js

```diff
@@ -0,0 +1,180 @@
+const classes = {
+    ACTIVE_LINK: 'active-link',
+    SIDE_BAR: 'side-bar',
+    SIDE_BAR_ACTIVE: 'side-bar-active',
+    HAMBURGER: 'hamburger',
+    SLIDE: 'slide',
+    PREV: 'prev',
+    INITIAL: 'initial',
+    NEXT: 'next',    
+    TAG: 'tag',
+    PORTFOLIO_EXAMPLES_CONTAINER: 'portfolio-examples-container',
+    PORTFOLIO_EXAMPLE: 'portfolio-example',
+    ACTIVE: 'active',
+}
+const selectors = {
+    SIDE_BAR_ACTIVE: '.side-bar-active',
+    SIDE_BAR: '.side-bar',
+    HAMBURGER: '.hamburger',
+    SLIDE: '.slide',
+    ARROW_RIGHT: '.arrow-right',
+    ARROW_LEFT: '.arrow-left',
+    TAG: '.tag',
+    PORTFOLIO_EXAMPLE: '.portfolio-example',
+    PORTFOLIO_EXAMPLES_CONTAINER: '.portfolio-examples-container',
+    DIV_HOME: 'div#home',
+    NAV_A: 'nav a',
+}
+const tags = {
+    HEADER: 'header',
+    SECTION: 'section',
+    FOOTER: 'footer',
+}
+const atTributes =  {
+    HREF: 'href',
+    ID: 'id',
+}
+const headerHeight = document.querySelector(tags.HEADER).offsetHeight;
+const portfolioFilters = document.querySelectorAll(selectors.TAG);
+
+let slides = document.querySelectorAll(selectors.SLIDE),
+    totalSlides = slides.length,
```

Comment:
you use everywhere totalSlides  - 1
suppose it will be better to store such value instead